### DISCUSSION TITLE
Vercel unable to import git repos

### DISCUSSION THREAD

**Discussion by theapeterson** [vercel/community#1427] [Help] [answered]
### Summary

My dashboard is stuck at the loading spinner and has not been able to import my git repos. Any idea how I can resolve this? Thank you. 
<img width="742" alt="Screen Shot 2023-02-02 at 1 28 38 PM" src="https://user-images.githubusercontent.com/119032990/216417962-535a0ce4-17a1-4450-8ff5-a57777a47f91.png">

**Comment by amyegan** [comment]
Thanks for reaching out @theapeterson. I have a few more questions to give me a better picture of what is going wrong:
- Is this what you see after recently enabling a [Git Integration](https://vercel.com/docs/concepts/deployments/git#) with a connected GitHub account?
- Is this a new problem, meaning your projects used to show up in the list and have stopped suddenly?
- Have you recently connected a new GitHub, GitLab, or Bitbucket account?
- Have you recently changed [Authorized GitHub App](https://github.com/settings/apps/authorizations) permissions for Vercel?

**Comment by theapeterson** [comment, marked as answer]
It happened after I added a team member.
It has been resolved since.
Thank you!

**Comment by batuhannon** [comment]
I've same problem here. I just sign up with my github . Dasboard stucked like that.

**Reply by amyegan** [reply]
Hey @batuhannon. Please [start a new discussion](https://github.com/vercel/community/discussions/new) with more information about your account and situation, or [submit a case](https://vercel.com/help#issues) to the Customer Success Team.

**Comment by matt-chaib** [comment]
I'm also having the same issue

**Reply by amyegan** [reply]
Hey @matt-chaib. Please [start a new discussion](https://github.com/vercel/community/discussions/new) with more information about your account and situation, or [submit a case](https://vercel.com/help#issues) to the Customer Success Team.
